guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder.adoc

```diff
@@ -0,0 +1,28 @@
+common:header-top.adoc[]
+
+== Sample Project
+
+You can link:@sourceDir@.zip[download a sample application] with the code examples in this article.
+
+== Introduction
+
+While developing web applications, we often need to build URIs. For example, to create redirection responses. In this article, we compare two similar APIs - Spring
+https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/UriComponentsBuilder.html[`UriComponentsBuilder`] and Micronaut https://docs.micronaut.io/latest/api/io/micronaut/http/uri/UriBuilder.html[UriBuilder].
```

Comment:
```suggestion
While developing web applications, we often need to build URIs. For example, to create redirection responses. In this article, we compare two similar APIs - Spring's
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/UriComponentsBuilder.html[`UriComponentsBuilder`] and Micronaut's https://docs.micronaut.io/latest/api/io/micronaut/http/uri/UriBuilder.html[UriBuilder].
```

Reply:
e cannot use Micronaut as a noun if we want to follow the brand guidelines

Reply:
but you used it as noun in the original text? All Alvaro added was the possessive